config/settings.toml

```diff
@@ -43,3 +43,5 @@ _remove_workspace = true
 
 # You will only need to edit the config relating to your hardware provider
 dynaconf_include = ["libvirt.toml", "openstack.toml", "aws_ec2.toml", "rook_upstream.toml", "ses.toml"]
+
+ansible_extra_vars = ""
```

Comment:
Could you please add a comment to document what this is expecting.

We should probably also call it ansible_extra_vars_file.

We should also make it either an absolute path or relative to rookcheck's dir.

Reply:
Also could you please move this line above the dynaconf_include (so that remains the last of settings.toml)

Reply:
Fixed